cmake/cross-toolchain-mingw32.cmake

```diff
@@ -2,8 +2,7 @@
 set( CMAKE_SYSTEM_NAME Windows )
 set( CMAKE_SYSTEM_PROCESSOR x86 )
 
-# C/C++ compilers
-set( CMAKE_C_COMPILER i686-w64-mingw32-gcc )
```

Comment:
I don't think it helps anything to provide an incomplete version of the toolchain file.

Reply:
This tool only use C++, I modified the CMake file to do:

```
project(${CRUNCH_PROJECT_NAME} LANGUAGES CXX)
```

Meaning that even CMake would not look for a C compiler to begin with.

Reply:
```
-- The CXX compiler identification is GNU 12.0.0
-- Detecting CXX compiler ABI info
-- Detecting CXX compiler ABI info - done
-- Check for working CXX compiler: /usr/bin/x86_64-w64-mingw32-g++ - skipped
-- Detecting CXX compile features
-- Detecting CXX compile features - done
-- Found Git: /usr/bin/git (found version "2.40.1") 
-- Performing Test CMAKE_HAVE_LIBC_PTHREAD
-- Performing Test CMAKE_HAVE_LIBC_PTHREAD - Success
-- Found Threads: TRUE  
-- Configuring done (0.6s)
-- Generating done (0.0s)
```

Reply:
I think of it like a library component that doesn't need to modified for the application's use case. Like when we have a vendored library, we don't go in and delete all the functions that we don't use.

Reply:
OK, I reverted that line.